<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Benchmark</class>
 <widget class="QDialog" name="Benchmark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dictionary size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dictSizeComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.2 MB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5 MB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25 MB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32 MB</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="chooseTestFileButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a custom file for testing</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="testFileLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="huffmanHeaderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Huffman algorithm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lz4HeaderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LZ4</string>
   </property>
  </widget>
  <widget class="QLabel" name="paq9aHeaderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Paq9a</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="huffmanCompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decompression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="huffmanDecompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression ratio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="huffmanCompressionRatioLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0 %</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lz4CompressionRatioLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0 %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lz4CompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decompression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lz4DecompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression ratio:</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="paq9aDecompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decompression speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="paq9aCompressionSpeedLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 MB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression ratio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="paq9aCompressionRatioLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0 %</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>610</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelCustomButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Use a built-in dictionary instead</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start benchmark</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
